py3status/modules/xfce4_notifyd.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+"""
+Display and toggle the DND (do-not-disturb) status of xfce4-notifyd
+
+Toggles DND status on click
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module (default 5)
+    dnd_off: status string when DND is off (default 'Off')
+    dnd_on: status string when DND is on (default 'On')
+    format: display format for this module (default 'DND: {status}')
```

Comment:
We could put all three in the `format`; new format looks like `(default 'DND: [\?if=status On|Off]')`. I think it would look more nice if the format could behave like `DPMS` by only displaying `DND` with either color; new format looks like `(default '\?color=status DND')`.

(Opinion) I wonder if `cache_timeout = 5` is too frequent for something that may not get changed much externally. It could be at `15` like `dpms`. Toggling would also update it immediately.

Reply:
I took `cache_timeout = 5` from the `systemd` module, which is in fact doing something very similar (polling the state of a dbus object). It would also be possible to set this up to listen for events to get told right away when the value changes - I pursued this for a while but it involves introducing a glib main loop into the module, which seemed like a bad idea.

Reply:
I will make the format change you suggest... (note that this would be another obstacle to merging with `do_not_disturb`, as this would be a backwards-incompatible change with it).